en/services/manual_control.md

```diff
@@ -0,0 +1,88 @@
+# Manual Control Protocol
+
+The Manual Control Protocol enables controlling a system using standard joystick axes nomenclature, along with a joystick-like input device. 
+ 
+The protocol is implemented with just the [`MANUAL_CONTROL`](../messages/common.md#MANUAL_CONTROL) message.
+It consists of the `target` system to be controlled, four primary axes (`x`, `y`, `z`, `r`), two extension axes (`s`, `t`), and two 16-bit fields to represent the states of up to 32 buttons (`buttons`, `buttons2`).
+Unused axes can be disabled, and the extension axes must be explicitly enabled using bits 0 and 1 of the `enabled_extensions` field.
+
+## Mapping Axes
+
+Manual control is performed in the vehicle-frame. All axis values are normalised to the range -1000 to 1000.
+
+### Rotation-Focused Control
+
+The typical axis assignments for a thrust- and rotation-controlled vehicle (e.g. planes, multicopters) are listed below.
+
+field | motion axis | +ve direction | -ve direction
+-- | -- | -- | --
+`x` | pitch | forward/nose-down | backward/nose-up
+`y` | roll | right-down | left-down
+`z` | thrust | positive | negative
+`r` | yaw | counter-clockwise | clockwise
+
+### Directional Control
+
+Vehicles with direct control over vehicle translation directions (multicopters) typically use the following mappings.
+
+field | motion axis | +ve direction | -ve direction
+-- | -- | -- | --
+`x` | forward | forward | backward
+`y` | lateral | right | left
+`z` | vertical | up | down
+`r` | yaw | counter-clockwise | clockwise
+`s` | pitch | forward/nose-down | backward/nose-up
+`t` | roll | right-down | left-down
+
+## Mapping Buttons
+
+Button functions are vehicle-specific and determined in the vehicle firmware.
+They are generally user-configurable using firmware parameters (e.g. ArduCopter's [`BTN_FUNCn`](https://ardupilot.org/copter/docs/parameters.html#btn-func1-button-pin-1-rc-channel-function) or ArduSub's [`BTNn_FUNCTION`](https://www.ardusub.com/developers/full-parameter-list.html#btnnfunction-function-for-button)), through the [Parameter](./parameter.md) or [Extended Parameter](./parameter_ext.md) protocols.
+
+The `buttons` field is required, and corresponds to the first 16 buttons.
+
+`buttons2` is an [extension](https://mavlink.io/en/guide/define_xml_element.html#message_extensions), and corresponds to the optional second set of 16 buttons.
+
+## Future Extensions
```

Comment:
My preference is that we omit this section - it complicates the story for end users to no great benefit.

Reply:
I feel like it could be useful information to people looking into what they should implement if they're making a new vehicle type, or just trying to understand reasoning about why the message is implemented the way it is and how that compares to other messages with similar functions. Not particularly fussed either way though - if it's more confusing/complicating than helpful then it should be removed (or perhaps just moved elsewhere).

Curious what others think on this.

Reply:
I put the simple design part in the introduction, and the future extensions is just a line at the very bottom of the doc. Compromise :-)